same-origin-policy/same-origin-policy.markdown

```diff
@@ -745,7 +745,7 @@ assert Confidentiality {
 }
 ```
 
-The integrity property is the dual of confidentiality: 
+The integrity property is the double of confidentiality: 
```

Comment:
In this context, we mean "integrity" is a concept that mirrors "confidentiality", so "dual" is more appropriate.